src/renderer/components/app.tsx

```diff
@@ -177,7 +177,7 @@ export class App extends React.Component {
       }
     });
 
-    for (const menu of this.extensionRoutes.keys()) {
```

Comment:
Why was this changed? It is both slower and introduces and more verbose?

Reply:
Wrong merge, you're faster then my revert-fix :D

Reply:
lol